tests/unit/lib/intrinsic_resolver/test_intrinsic_resolver.py

```diff
@@ -527,7 +530,7 @@ def test_fn_sub_second_argument_invalid_formats(self, name, intrinsic):
 
     @parameterized.expand(
         [
-            ("If Fn::Sub is a list, it should only have 2 arguments".format(item), item)
+            (f"If Fn::Sub is a list, it should only have 2 arguments {', '.join(item)}", item)
```

Comment:
Why the join here but `{item}` a few lines up? If it doesn't matter I'd go with less code.

Reply:
yeah it does not matter, I can remove the join, sounds better